lib/class-wp-theme-json-gutenberg.php

```diff
@@ -981,8 +972,25 @@ private function get_block_classes( $style_nodes ) {
 			$selector     = $metadata['selector'];
 			$settings     = _wp_array_get( $this->theme_json, array( 'settings' ) );
 			$declarations = self::compute_style_properties( $node, $settings );
+
+			// 1. Separate the ones who use the general selector and the ones who have th
```

Comment:
Looks like this comment isn't finished

Reply:
Fixed at https://github.com/WordPress/gutenberg/pull/35318/commits/a7e3e8f08512cac3ec12c45a12d13e7e4c6e0408